experimental/trusted_shuffler/backend/src/main.rs

```diff
@@ -36,12 +39,24 @@ pub struct Opt {
     listen_address: String,
 }
 
+lazy_static! {
+    static ref START_TIME: Instant = Instant::now();
```

Comment:
I'm curious whether this actually works? I thought it is initialized the first time it is accessed (hence the laziness), not really at "static initialization" time (which I don't think it's really something that exists). But curious to know either way.

Reply:
I think START_TIME is only initialized when you first call it, which is also when you call elapsed on it, so I imagine you would get almost zero elapsed here the first time you invoke it. But I see your point, since you are only using it for relative offsets, right?

Reply:
yes, only relative offsets!

Reply:
Could you ellaborate on the reason for using a static instance rather than creating a new `Instant` for every request? I think this might actually be slower due to the locking involved with using a single static instance.

Reply:
It seems simpler to just use the actual local instant everywhere, I mean that's what it does already, right? It gives you the number of seconds since epoch.